elements/pfe-avatar/src/pfe-avatar.scss

```diff
@@ -1,35 +1,38 @@
 @import "../../pfe-sass/pfe-sass";
 
-:host {
-  --pfe-avatar--width: 128px;
+$LOCAL: avatar;
+$LOCAL-VARIABLES: (
+  size: 128px
```

Comment:
I originally came to this branch with a mindset to tidy up variables for 1.0 which is where a few of the renames happened along the way.  What would you think of my building in a backwards-compatibility map:

```
// TODO: This will be removed at 1.0
$backwards-compatibility: (
    width: pfe-local(size)
);

$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
```